src/agoradatatools/etl/transform/rna_distribution.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+
+
+def transform_rna_seq_data(datasets: dict):
```

Comment:
Agreed, this should be in its own file.

Reply:
I guess I thought that since `transform_rna_seq_data` is used inside of `transform_rna_distribution_data` having them together made sense, but I will go ahead and make this change before I merge.

Reply:
Oooh that's tricky now that you point it out. We actually use `transform_rna_seq_data` on its own as a separate transform that generates its own JSON file, but then when we want RNA distribution data in a second transform we call `transform_rna_seq_data` again from `transform_rna_distribution_data` because we need the same transformed dataset. We currently do it this way because previously, we'd call `transform_rna_seq_data`, write the JSON file, and then `transform_rna_distribution_data` depended on the JSON file we just created and that seemed too fragile. 
But yeah, hmmm. Maybe this is a case of "make a ticket about it and figure it out later"?

Reply:
Yeah, I guess I will hold off. Maybe when we go to implement the tests on these transforms it will make whether they should be together or not more clear

Reply:
We already have AG-1097 for this one, feel free to add any brilliant ideas to it! I don't think it's a problem to leave it as-is for now.